mlir/test/lib/Interfaces/TilingInterface/lit.local.cfg

```diff
@@ -0,0 +1 @@
+config.suffixes.remove(".td")
```

Comment:
do we need the file?

Reply:
Yeah, without this the lit test framework expects the .td file added here to contain tests (not sure why). This avoids it.